improver_tests/psychrometric_calculations/cloud_condensation_level/test_MetaPluginCloudCondensationLevel.py

```diff
@@ -0,0 +1,35 @@
+# (C) Crown copyright, Met Office. All rights reserved.
+#
+# This file is part of improver and is released under a BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+from unittest.mock import patch, sentinel
+
+from improver.psychrometric_calculations.cloud_condensation_level import (
+    MetaPluginCloudCondensationLevel,
+)
+
+
+@patch(
+    "improver.psychrometric_calculations.psychrometric_calculations.HumidityMixingRatio"
+)
+@patch(
+    "improver.psychrometric_calculations.cloud_condensation_level.CloudCondensationLevel"
+)
+def test_basic(mock_CloudCondensationLevel, mock_HumidityMixingRatio):
```

Comment:
This simply ensures that this meta-plugin, calls the underlying plugins in the way we intended.